lib/traits/slim/thermostat_slim_widget.dart

```diff
@@ -48,16 +48,26 @@ class ThermostatSlimWidget extends BaseSlimWidget {
                               ?.copyWith(
                                   color: WidgetStyleConstants.darkTextColor),
                         ),
-                        Text(
-                            thermostatProvider.getAvailableFanModes
-                            .map((mode) => mode.value)
-                                .join(', '),
-                            style: Theme.of(context)
-                                .textTheme
-                                .headline6
-                                ?.copyWith(
-                                    color: WidgetStyleConstants.darkTextColor,
-                                    fontWeight: FontWeight.normal))
+                        ...List<Widget>.generate(
+                          thermostatProvider.getAvailableFanModes.length,
```

Comment:
Just a thought - should we be calling this thermostatProvider? The trait is called ThermostatSetting.  As a general rule, wouldn't it make sense to name them consistently with the traits they are intended for? e.g. thermostatSettingProvider.  Thoughts?  cc @surajadh @ses110

Reply:
Closing this pull request but we definitely could do better on a number of naming things throughout. We have a mix of stuff going on and should probably solidify on some conventions.